aptible/database.go

```diff
@@ -191,19 +197,34 @@ func (c *Client) UpdateDatabase(databaseID int64, updates DBUpdates) error {
 		request.DiskSize = updates.DiskSize
 	}
 
-	params := operations.NewPostDatabasesDatabaseIDOperationsParams().WithDatabaseID(databaseID).WithAppRequest(&request)
-	op, err := c.Client.Operations.PostDatabasesDatabaseIDOperations(params, c.Token)
-	if err != nil {
-		return err
+	if !updates.OnlyChangingHandle {
```

Comment:
by default it is false, default path and entry point is here. the only time this should get hit is if someone is updating an app BESIDES its handle